src/core/ext/xds/xds_client_grpc.cc

```diff
@@ -115,11 +115,20 @@ const auto kMetricResourceUpdatesInvalid =
          kMetricLabelXdsResourceType},
         {}, false);
 
+const auto kMetricServerFailure =
+    GlobalInstrumentsRegistry::RegisterUInt64Counter(
+        "grpc.xds_client.server_failure",
+        "EXPERIMENTAL.  A counter of xDS servers going from healthy to "
+        "unhealthy.  A server goes unhealthy when we have a connectivity "
+        "failure or when the ADS stream fails without seeing a response "
+        "message, as per gRFC A57.",
+        "{failure}", {kMetricLabelTarget, kMetricLabelXdsServer}, {}, false);
+
 const auto kMetricConnected =
     GlobalInstrumentsRegistry::RegisterCallbackInt64Gauge(
         "grpc.xds_client.connected",
         "EXPERIMENTAL.  Whether or not the xDS client currently has a "
-        "working ADS stream to the xDS server. For a given server, this "
+        "working ADS stream to the xDS server.  For a given server, this "
```

Comment:
probably unintended change

Reply:
It was intentional.  I'm firmly in the "two spaces after a period" camp.  Must have missed this in my earlier PR.

Reply:
wait, what? I didn't know this was a thing..